lib/internal/modules/esm/loader.js

```diff
@@ -343,24 +382,21 @@ class CustomizedModuleLoader extends DefaultModuleLoader {
    * @param {object} [context] Metadata about the module
    * @returns {Promise<{ format: ModuleFormat, source: ModuleSource }>}
    */
-  async load(url, context) {
-    const result = await hooksProxy.makeAsyncRequest('load', url, context);
-    this.validateLoadResult(url, result?.format);
```

Comment:
Is this validation no longer happening?

Reply:
It is, it's just combined as part of the `DefaultModuleResolver`'s `load` method here so that "delegates" don't ever have to call it manually: https://github.com/nodejs/node/pull/48559/files#diff-f6796082f599554ec3a29c47cf026cb24fc5104884f2632e472c05fe622d778bR316